packages/terra-application-header-layout/translations/fr.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
We need `fr-FR ` translations as well

Reply:
https://github.com/cerner/terra-framework/pull/301/commits/5c99136fb9295b0edf9e4b6da11ad8d01d4c7b57